index.html

```diff
@@ -1,4 +1,11 @@
 <html style="color-scheme: light dark">
 <center>
-  <h1 style="font-size: 120px">Yes.</h1>
+  <h1 id="statusText" style="font-size: 120px">Yes.</h1>
+  <script>
+      if(navigator.connection.type == "wifi") {
```

Comment:
```suggestion
      if((navigator.connection || {}).type === "wifi") {
```
For browsers without `navigator.connection` :shrug:

Reply:
Thanks!